storybook/src/Models/ManageCollectiblesModel.qml

```diff
@@ -0,0 +1,153 @@
+import QtQuick 2.15
+import QtQml.Models 2.15
+
+import Models 1.0
+
+ListModel {
+    function randomizeData() {
+        // TODO
+    }
+
+    readonly property var data: [
+        {
+            uid: "fp#9140",
+            name: "Frenly Panda #9140",
+            collectionUid: "",
+            collectionName: "",
+            communityId: "fpan",
+            communityName: "Frenly Pandas",
+            communityImage: "https://pbs.twimg.com/profile_images/1599347398769143808/C6qG3RQv_400x400.jpg",
+            imageUrl: "https://i.seadn.io/gae/qPfQjj4P1w0xVQXAmQJLmQ4ZtLFAJU6oiH69Lsny82LFbipLAgXhHKrcLBx2U09SmRnzeHY0ygz-3NIb-JegE_hWrZquFeL-qUPXPdw",
```

Comment:
1. Referring to some external resources is a bit risky, the page may become unavailable, the content may change and copyrights may be violated. So ideally we should refer to some owned resources.
2. We keep some longer resources in ModelData, maybe putting some images here as base64 would be a solution.

Reply:
Hmmm... it's definitely not a (c) violation, it would be if we included it which I didn't want (in order not to bloat the already big sources). True that it might disappear any time bug really not a big deal imo

Reply:
As you prefer. I'm just pointing out that there is already a certain convention for this type of thing.